docs/TektonConfig.md

```diff
@@ -550,6 +550,11 @@ options:
               averageUtilization: 85
               type: Utilization
           type: Resource
+  webhookConfigurationOptions:
+    validation.webhook.pipeline.tekton.dev:
+      failurePolicy: Fail
+      timeoutSeconds: 20
+      sideEffects: None
```

Comment:
I see webhookConfigurationOptions is supported for both Validation and Mutation can we update this example to show that as well

Reply:
done

Reply:
it does not make a difference, at the end it is just name